src/components/header/header_links/header_links.tsx

```diff
@@ -95,16 +95,15 @@ export const OuiHeaderLinks: FunctionComponent<OuiHeaderLinksProps> = ({
   popoverProps,
   ...rest
 }) => {
-  const { onClick: _onClick, iconType = 'apps', ...popoverButtonRest } = {
-    ...popoverButtonProps,
-  };
+  const { onClick, iconType = 'apps', ...popoverButtonRest } =
+    popoverButtonProps || {};
```

Comment:
Can we use the nullish coalescing operator here instead of logical or?

Reply:
Since this prop can be either only undefined or only an object, it would be better to use `??`. But it looks good with `||` as well

Reply:
Fixed by `||` => `??`